### PR TITLE
[CDX-222] Bugfix: Slider range not rendering correctly if facet.status.min === 0

### PR DESCRIPTION
**Pull request by Mudaafi**
# Description
The range slider was not rendering correctly if `facet.status.min === 0`. This is due to a logical check failing where it's not supposed to.

# Pull Request Checklist

Before you submit a pull request, please make sure you have to following:
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] I have made sure my PR is up-to-date with the main branch.

# PR Type

What kind of change does this PR introduce?

- [x] Bugfix

### CODE REVIEW

File: spec/components/Filters/Filters.test.jsx [hunk 1]
```diff
@@ -211,6 +211,55 @@ describe('Testing Component: Filters', () => {
         expect(maxInputSlider.value).toBe(mockPriceFacet.status.max.toString());
       });
     });
+
+    it.only('Edge Case: If facet.status.min = 0, should render ranged filters that have already been applied correctly', async () => {
```
Comment (stanlp1) [inline spec/components/Filters/Filters.test.jsx:215]: Only

**Review by stanlp1 — APPROVED** [review]
One comment .LGTM

File: spec/components/Filters/Filters.test.jsx (hunk 1 above)
Comment (Mudaafi) [reply spec/components/Filters/Filters.test.jsx:215]: thanks for catching that!
